star-ros2/scripts/test_gateway_only.sh

```
@@ -0,0 +1,101 @@
+echo "  Started (PID: $GATEWAY_PID)"
+sleep 3
+
+echo ""
+echo "Running for $DURATION_SECONDS seconds..."
+echo ""
+
+START_TIME=$(date +%s)
+while true; do
+    ELAPSED=$(($(date +%s) - START_TIME))
+
+    if [ $ELAPSED -ge $DURATION_SECONDS ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)